src/docfx/lib/MaskUtility.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Linq;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Docs.Build
+{
+    internal static class MaskUtility
```

Comment:
move to `StringUtility`?

Reply:
It also handles JToken, mask comes more reasonable